### DISCUSSION TITLE
nfqws и WireGuard

### DISCUSSION THREAD

**Discussion by x12lab** [Anonym-tsk/nfqws-keenetic#73] [Q&A]
Можно ли донастроить nfqws для обхода блокировки протокола WireGuard? Что нужно добавить в конфиг, кроме собственно адреса сервера и порта?

**Comment by statgit** [comment]
Вам лучше поставить WG от amnezia (можно на сам роутер keenetic), там уже реализовано все что бы не палить стандартные сигнатуры wireguard.

**Reply by whitemonster88** [reply]
amnezia это немного не то. Он конечно работает, но не так хорошо, как обычный wireguard (уебищный клиент)

**Reply by x12lab** [reply]
Мне нужно подключиться к рабочему VPN серверу, который обычный WireGuard. Если воткнуть на роутер клиент от амнезии, оно будет работать? И как оно в плане одновременной работы с nfqws? Я просто не интересовался, не в курсе. Мысль была такая, что посредством nfqws можно точно так же подхватывать пакеты wireguard и менять их, как и пакеты https. Но семья и работа занимают все ресурсы, едва жить успеваю, поэтому никак руки не дойдут посидеть и самому поразбираться с настройками, чтобы попробовать воплотить идею на практике.
